File: 04_ruby/d02/Tim_Rourke/morning_ex/turtles.js
function Turtle(name, color, weapon, saying) {
	this.name = name;
	this.color = color;
	this.weapon = weapon;
	this.saying = saying;
	this.sayStuff = function(){
		console.log(this.saying);
	}
}

var raphael = new Turtle('Raphael', 'red', 'pointy sticks', 'I\'m an asshole');

var michelangelo = new Turtle('Michelangelo', 'orange', 'dogshit', 'BOOOOOOOOOOOOOOP!');

var tinyWhiny = new Turtle('Peewee', 'poypul', 'feelings', 'snurp');

raphael.sayStuff();

File: 04_ruby/d02/Tim_Rourke/morning_ex/turtles.rb
class Turtle
	def initialize(name, color, weapon, saying)
		@name = name
		@color = color
		@weapon = weapon
		@saying = saying
	end

	def name
		@name
	end

	def color
		@color
	end

	def weapon
		@weapon
	end

	def saying
		@saying
	end

	def say_stuff
		puts @saying
	end
end

timmy = Turtle.new('Tim', 'red', 'sticks', 'Shut up dude')

puts timmy.color
timmy.say_stuff
